lib/src/header/header.dart

```diff
@@ -453,52 +453,59 @@ class ClassicalHeaderWidgetState extends State<ClassicalHeaderWidget>
 
   /// 文本
   String get _refreshText {
-    return widget.classicalHeader.refreshText ??
-            GlobalEasyRefreshLocalizations.of(context) == null
-        ? _localizations.pullToRefresh
-        : GlobalEasyRefreshLocalizations.of(context).pullToRefresh;
+    return widget.classicalHeader.refreshText != null
+        ? widget.classicalHeader.refreshText
+        : GlobalEasyRefreshLocalizations.of(context) == null
+            ? _localizations.pullToRefresh
+            : GlobalEasyRefreshLocalizations.of(context).pullToRefresh;
   }
 
   String get _refreshReadyText {
-    return widget.classicalHeader.refreshReadyText ??
-            GlobalEasyRefreshLocalizations.of(context) == null
-        ? _localizations.releaseToRefresh
-        : GlobalEasyRefreshLocalizations.of(context).releaseToRefresh;
+    return widget.classicalHeader.refreshReadyText != null
+        ? widget.classicalHeader.refreshReadyText
+        : GlobalEasyRefreshLocalizations.of(context) == null
+            ? _localizations.releaseToRefresh
+            : GlobalEasyRefreshLocalizations.of(context).releaseToRefresh;
   }
 
   String get _refreshingText {
-    return widget.classicalHeader.refreshingText ??
-            GlobalEasyRefreshLocalizations.of(context) == null
-        ? _localizations.refreshing
-        : GlobalEasyRefreshLocalizations.of(context).refreshing;
+    return widget.classicalHeader.refreshingText != null
+        ? widget.classicalHeader.refreshingText
+        : GlobalEasyRefreshLocalizations.of(context) == null
+            ? _localizations.refreshing
+            : GlobalEasyRefreshLocalizations.of(context).refreshing;
   }
 
   String get _refreshedText {
-    return widget.classicalHeader.refreshedText ??
-            GlobalEasyRefreshLocalizations.of(context) == null
-        ? _localizations.refreshed
-        : GlobalEasyRefreshLocalizations.of(context).refreshed;
+    return widget.classicalHeader.refreshedText != null
```

Comment:
return widget.classicalHeader.refreshedText != null
        ? widget.classicalHeader.refreshedText
        : GlobalEasyRefreshLocalizations.of(context) == null
            ? _localizations.refreshed
            : GlobalEasyRefreshLocalizations.of(context).refreshed;